generated/beats/fields.ecs.yml

```diff
@@ -10167,7 +10171,6 @@
       A span represents an operation within a transaction, such as a request to another
       service, or a database query.'
     example: 3ff9a8981b7ccd5a
-    default_field: false
```

Comment:
Does it matter if `default_field` remains defined on a root field when it's `false`?

Reply:
It can be explicitly defined as false, but is unnecessary as Filebeat defaults to false, that's why the generator doesn't include it.